### PR TITLE
Cost Expandable Meta Changelog

### PR DESCRIPTION
**Pull request by marlenc3**

### CODE REVIEW

File: fern/docs/content/change-log/2024-09.mdx [hunk 1]
```diff
@@ -2,6 +2,29 @@
 title: Changelog | September, 2024
 ---
 
+## New Expandable Meta: Cost ##
+
+You can now request your cost data to be returned via API or when using our SDK,
+allowing for better insight on your prompt's economics.
+This works similar to requesting usage data for your requests like so:
+
+```
```
Comment (m-abboud) [inline fern/docs/content/change-log/2024-09.mdx:11]:
```suggestion
```json
```

File: fern/docs/content/change-log/2024-09.mdx [hunk 2]
```diff
@@ -2,6 +2,29 @@
...
+"expand_meta" : {
+    "cost": true
+}
+```
+You can expect a corresponding value to be passed in the meta fields on a response.
+```
```
Comment (m-abboud) [inline fern/docs/content/change-log/2024-09.mdx:17]:
```suggestion
```json
```

**Review by m-abboud — APPROVED** [review]

File: fern/docs/content/change-log/2024-09.mdx (hunk 1 above)
Comment (marlenc3) [reply fern/docs/content/change-log/2024-09.mdx:11]: oh BLESS

**Comment by noanflaherty** [comment]
I'm going to push up some edits to this and include updates to our API docs, too

**Comment by marlenc3** [comment]
> I'm going to push up some edits to this and include updates to our API docs, too

@noanflaherty  go for it!

**Review by dvargas92495 — COMMENTED** [review]
✅

**Pull request merged**